fuse_examples/classification/MG_CMMD/dataset.py

```diff
@@ -157,16 +157,17 @@ def merge_clinical_data_with_dicom_tags(root_path: str, target: str) -> str:
     :return: the new csv file path
     """
     input_source = root_path + '/CMMD_clinicaldata_revision.csv'
-    combined_file_path = root_path + '/files_combined.csv'
+    combined_file_path = data_misc_dir + '/files_combined.csv'
     if os.path.isfile(combined_file_path):
         return combined_file_path
+    Path(data_misc_dir).mkdir(parents=True, exist_ok=True)
     clinical_data = pd.read_csv(input_source)
     scans = []
-    for patient in os.listdir(root_path + 'CMMD/'):
-        path = root_path + 'CMMD/' + patient
+    for patient in os.listdir(root_path + '/CMMD/'):
```

Comment:
use os.path.join instead of '+'
Try to modify other places you use + as well

Reply:
done